utils/yearContext.js

```
@@ -0,0 +1,32 @@
+    constructor(yearsIndex, index) {
+        this.yearsIndex = yearsIndex;
+        this.index = index
+    }
+    get year() {
+        return this.yearsIndex.years[this.index]
+    }
+    get previous() {
+        if (this.yearsIndex.years.length > 1) {
+            return new YearContext(this.yearsIndex, this.index - 1)
+        }
+    }
```

Biome (1.9.4)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)